docs/modules/opa/examples/getting_started/opa.yaml

```diff
@@ -4,8 +4,25 @@ kind: OpaCluster
 metadata:
   name: simple-opa
 spec:
+  clusterConfig:
+    userInfoFetcher:
+      backend:
+        keycloak:
+          url: http://192.168.122.1:8080
+          credentialsSecretName: opa-keycloak
+          adminRealm: master
+          userRealm: master
+          clientId: admin-cli
```

Comment:
Getting started should use none backend

Reply:
Not sure if we want to document this yet at all...

Reply:
Document the userinfofetcher, or "none" backend?

Reply:
The UIF. Obviously it should be documented *eventually*, it's more about at what stage we want to encourage people to start using it tbh.

Reply:
clusterConfig was removed in 61e58b2db3e3beefb759026fc1940f327ee35a85